CONTRIBUTING.md

```diff
@@ -80,3 +80,17 @@ Please provide information that helps the developer test any changes they make b
 ## Coding style guidelines
 
 Use idomatic go. Document exported functions.
+
+# Publishing Tagged Release to Homebrew
+
+ [Homebrew](https://brew.sh) is used to install `wsk` locally. Once we release a new version of `wsk` we should update its version in homebrew.
+
+ Get the new release SHA256 checksum by downloading the Source Code (tar.gz) from the [releases page](https://github.com/apache/incubator-openwhisk-cli/releases) and running `shasum -a 256 X.Y.Z.tar.gz` on the tarball.
+
+ Update brew formula with the automation command `brew bump-formula-pr`:
+ ```bash
+ $ brew bump-formula-pr \
+   --url='https://github.com/apache/incubator-openwhisk-cli/archive/X.Y.Z.tar.gz' \
```

Comment:
The URL https://github.com/apache/incubator-openwhisk-cli/archive is not a valid link.

Reply:
Hi @houshengbo, you have to provide a valid link there yourself taken from the [releases page](https://github.com/apache/incubator-openwhisk-cli/releases) for example for version 0.9.0: https://github.com/apache/incubator-openwhisk-cli/archive/0.9.0-incubating.tar.gz 

I'm happy to change the wording to make it more clear if you'd like.

Reply:
Yes, please make sure what is the valid link, what the user should change in order to make the link work. My suggestion is to have a link with a certain format, and take a certain version as an example to provide a valid link. 

Format: https://github.com/apache/incubator-openwhisk-cli/archive/<X.Y.Z.tar.gz>.
Example for 0.9.0 is https://github.com/apache/incubator-openwhisk-cli/archive/0.9.0-incubating.tar.gz
The user needs to give a valid link by changing the name of the the artifact.